InternshipBE/WebApi/Controllers/WeatherForecastController.cs

```diff
@@ -13,7 +13,7 @@ public class WeatherForecastController : ControllerBase
     {
         private static readonly string[] Summaries = new[]
         {
-            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+            "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering"
         };
```

Comment:
Why deleted?

Reply:
Because its my examble working with git

Reply:
ok you are getting a decline your pull request

Reply:
ok